zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -0,0 +1,87 @@
+from zigpy import types as t
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import (
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.manufacturer_specific import ManufacturerSpecificCluster
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.xiaomi import BasicCluster, XiaomiCluster, XiaomiCustomDevice
+
+
+class OppleCluster(XiaomiCluster, ManufacturerSpecificCluster):
+    """Add Opple cluster for power outage memory attribute."""
+
+    cluster_id = 0xFCC0
+    ep_attribute = "opple_cluster"
+    attributes = {
+        0x0201: ("power_outage_memory", t.Bool, True),
+    }
+
+
+class LumiLightAcn003(XiaomiCustomDevice):
```

Comment:
Maybe use just `CustomDevice` here. I don't think it really matters, but `XiaomiCustomDevice` is mostly used for specifying a variable used for battery size.
(and I don't think this is a battery-powered lamp?)

Reply:
I would keep it because the name `XiaomiCustomDevice` doesn't imply it is for battery powered devices only, it sounds more like generic class for xiaomi devices also we might need to add some extra logic for non-battery devices and this will be a good place. 

Current logic for the battery shouldn't make any harm to mains powered devices anyway. WDYT?